### PR TITLE
[PHP] Add CodeIgniter 4 Framework

### LINKED ISSUE #3124: [PHP] Add CodeIgniter4
Hi @waghanza, can you add [CodeIgniter4](https://github.com/codeigniter4/CodeIgniter4)?

### PR DESCRIPTION
**Pull request by michalsn**
----
Closes #3124

### CODE REVIEW

**Comment by michalsn** [comment]
Yes, sure.
I'm just a collaborator for CodeIgniter4, but yes (maybe we will change the assigned person later).

**Comment by waghanza** [comment]
@michalsn the results has been updated in https://github.com/the-benchmarker/web-frameworks#results. they include `codeigniter4`

File: php/codeigniter4/config.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+framework:
+  website: codeigniter.com
+  version: 4.0.3
```
Comment (waghanza) [inline php/codeigniter4/config.yaml:4]: please us X.Y`

File: php/codeigniter4/composer.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+	"require": {
+		"codeigniter4/framework": "^4"
```
Comment (waghanza) [inline php/codeigniter4/composer.json:3]: please use `~4.0.0`

**Pull request merged**